concent_api/core/tests/test_unit_validation.py

```diff
@@ -18,6 +21,15 @@
 from core.validation import validate_secure_hash_algorithm
 from core.validation import validate_subtask_price_task_to_compute
 from utils.constants import ErrorCode
+from utils.testing_helpers import generate_ecc_key_pair
+from utils.testing_helpers import generate_priv_and_pub_eth_account_key
+
+
+REQUESTOR_PUBLIC_KEY = generate_ecc_key_pair()[1]
+REQUESTOR_ETH_ADDRESS = generate_priv_and_pub_eth_account_key()[1]
```

Comment:
You have this available in `ConcentIntegrationTestCase` in `self.`.

Reply:
fixed